libs/components/filter-bar/src/lib/modules/filter-bar/filter-bar.component.scss

```diff
@@ -11,3 +11,12 @@
     var(--sky-comp-filter_bar-space-inset-bottom)
     var(--sky-comp-filter_bar-space-inset-left);
 }
+
+// When filter bar is a child of data manager toolbar, add a top border as if it were a toolbar section.
+:host-context(.sky-data-manager-toolbar) {
+  border-top: var(
+    --sky-override-toolbar-border-top-bottom,
+    var(--sky-border-width-divider) var(--sky-border-style-divider)
+      var(--sky-color-border-divider)
+  );
+}
```

Comment:
So this visual look is the same as the `border-top` from toolbar for a toolbar section, can we get a visual review from design that that is what we want here? Or is there a comment or discussion somewhere on how we landed on this styling?

For that to happen we would also need a visual test for this and it looks like none of the existing visual tests for toolbar were change from this PR, can we add a visual test?